testing/vault/vault.pre-deinstall

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Stop vault service
+rc-service vault stop &> /dev/null
```

Comment:
Well if they don't stop it and they have it running then it keeps on running and then if it's killed and added back again "service vault status" shows crashed. I could update this not to be a bashism.

This one was also copied from consul. Maybe another thing to fix there.